tools/catalog-export/src/loaders/tiger/tigerDateDatasets.ts

```diff
@@ -1,35 +1,30 @@
 // (C) 2007-2020 GoodData Corporation
 
 import { Attribute, DateDataSet } from "../../base/types";
-import { AttributeResourceSchema, DatasetResourceSchema, ITigerClient } from "@gooddata/api-client-tiger";
-import { DefaultGetOptions } from "./tigerClient";
+import { Attributes, AttributesItem, DatasetsItem, ITigerClient } from "@gooddata/api-client-tiger";
 import {
     convertAttribute,
-    convertTags,
     createDatasetMap,
     createLabelMap,
-    createTagMap,
     DatasetMap,
     getReferencedDataset,
     LabelMap,
-    TagMap,
 } from "./tigerCommon";
 
 type DatasetWithAttributes = {
-    dataset: DatasetResourceSchema;
-    attributes: AttributeResourceSchema[];
+    dataset: DatasetsItem;
+    attributes: AttributesItem[];
```

Comment:
Here you replaced `AttributeResourceSchema[]` by `AttributesItem[]` but on line 20, you replaced it by `Attributes`. This should probably stay consistent, right?

Reply:
it's valid as the returned type is `Attributes` and next logic iterates on the objects inside (`AttributesItem`)